ras_backstage/decorators/jwt_decorators.py

```diff
@@ -0,0 +1,20 @@
+from functools import wraps
+import logging
+
+from structlog import wrap_logger
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_jwt(request):
+    def extract_session(original_function):
+        @wraps(original_function)
+        def extract_session_wrapper(*args, **kwargs):
+            encoded_jwt_token = request.headers.get('Authorization')
+            if encoded_jwt_token:
+                return original_function(encoded_jwt_token, *args, **kwargs)
+            else:
+                raise Exception
```

Comment:
Can we add a more specific exception here? BadRequest or ApiError? and then handle it with a error handler.

Reply:
In fronstage-api we have a JWT-error which raises a 403 I think, I can add this here too.

I think originally I left it out to try and stop this PR getting too big but I think we're passed that point now...

Reply:
yh that sounds good